test/integration-tests/utils/common.ts

```diff
@@ -4,7 +4,7 @@ import fs = require("fs")
 
 function readYml (path: string) {
   try {
-    return yaml.safeLoad(fs.readFileSync(path, "utf8"));
+    return yaml.load(fs.readFileSync(path, "utf8"));
```

Comment:
Why change this?

Reply:
`safeLoad` deprecated with yaml version upgrade
Use the proper one